src/renderer/pages/Query/QueryAction.ts

```diff
@@ -17,7 +17,17 @@ const QueryAction = {
       Database.Chart.getAll()
```

Comment:
Almost all #143 code is exists.
I move dataSourceWithTables to member of QueryAction class, not to call while app is launching.